test/e2e/make_requests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to send requests to a Kubeflow metadata server to
+# verify it works.
```

Comment:
What does this test btw? That curl calls succeed? Should we care about the response itself?

Reply:
Now I just manually check it returns 200. It could be later automated to test whether 200 and expected response are returned.